assets/timber.js.liquid

```diff
@@ -127,6 +124,19 @@ timber.accessibleNav = function () {
   }
 };
 
+timber.drawersInit = function () {
+  timber.LeftDrawer = new timber.Drawers('NavDrawer', 'left');
```

Comment:
Encourage, but not make mandatory. It's better than most of the mobile nav solutions we've created so far, but wouldn't fit into every theme's style from my point of view.